### PR TITLE
LPS-156045

### PR DESCRIPTION
**Pull request by guivilarim**

### CODE REVIEW

**Comment by guivilarim** [comment]
ci:test:sf

**Comment by guivilarim** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+				Questions.tearDown();
+			}
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "This is a test for LPS-159138. Verify if the user is able to subscribe a tag if they want to."
+	@priority = "5"
+	test CanASubscribeAQuestion {
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase:45]: I would change the name from CanASubscribeAQuestion to CanSubscribeAQuestion, without the **A** from Can**A**

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+		}
+
+		task ("Validate the subcribed question") {
+			Questions.viewSubscribedQuestion(subscribedQuestion = "Question1");
+		}
+	}
+
+	@description = "This is a test for LPS-159144. Verify if the user is able to subscribe a tag if they want to."
+	@priority = "5"
+	test CanASubscribeATag {
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase:72]: I would change the name from CanASubscribeAQuestion to CanSubscribeAQuestion, without the A from CanA

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+		}
+
+		task ("Validate the subcribed tag") {
+			Questions.viewSubscribedTag(subscribedTag = "new");
+		}
+	}
+
+	@description = "This is a test for LPS-159160. Verify if the user is able to subscribe a topic if they want to."
+	@priority = "5"
+	test CanASubscribeATopic {
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/questions/QuestionsSubscriptionEnhanchement.testcase:108]: I would change the name from CanASubscribeAQuestion to CanSubscribeAQuestion, without the A from CanA

File: portal-web/test/functional/com/liferay/portalweb/macros/Questions.macro [hunk 4]
```diff
@@ -977,6 +977,20 @@ definition {
...
 	}
 
+	macro viewSubscribedTag {
+		if (isSet(subscribedTag)) {
+			AssertTextEquals(
+				key_subscribedTag = "${subscribedTag}",
+				locator1 = "Questions#SUBSCRIBED_TAG",
+				value1 = "${subscribedTag}");
+		}
+		else if (isSet(nosubscribedTag)) {
```
Comment (barrosdennys) [inline portal-web/test/functional/com/liferay/portalweb/macros/Questions.macro:987]: I would change to "noSubscribedTag"

**Comment by guivilarim** [comment]
ci:test:sf

**Comment by guivilarim** [comment]
ci:test:sf

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**